src/ol/Overlay.js

```diff
@@ -408,10 +408,10 @@ class Overlay extends BaseObject {
   /**
    * Pan the map so that the overlay is entirely visible in the current viewport
    * (if necessary).
-   * @param {PanIntoViewOptions|undefined} panIntoViewOptions Options for the pan action
+   * @param {PanIntoViewOptions|undefined} opt_panIntoViewOptions Options for the pan action
```

Comment:
```suggestion
   * @param {PanIntoViewOptions=} opt_panIntoViewOptions Options for the pan action
```

Reply:
This is the way we define optional arguments elsewhere. When you commit that, the CircleCI process will run again and hopefully return green.